src/components/atoms/Button/Button.tsx

```diff
@@ -108,29 +108,28 @@ const SButton = styled.button<IButtonProps>`
   cursor: pointer;
   border: 1px solid ${({ styling }) => backgroundColors[styling]};
   border-radius: 8px;
-  color: ${({ contrastColor }) => contrastColor};
-  color: ${({ styling }) => fontColors[styling]};
+  color: ${({ styling, contrastColor }) => (contrastColor ? contrastColor : fontColors[styling])};
   background-color: ${({ styling }) => backgroundColors[styling]};
   transition: all 140ms ease-in-out;
 
-  &:enabled {
+  &:not(:disabled) {
     box-shadow: ${({ styling }) => boxShadows[styling] || 'none'};
```

Comment:
Wouldn't it be easier to just move that box-shadow outside of the enabled? Or am I missing something?

Reply:
I tried few months ago to not use actual `border` instead of faking it with a `box-shadow` 🙄 but got tired and left as it was 😪 😅